packages/ui/src/exports/elements.ts

```diff
@@ -48,7 +48,11 @@ export { UnpublishMany } from '../elements/UnpublishMany/index.js'
 export { Upload } from '../elements/Upload/index.js'
 export { BlocksDrawer } from '../fields/Blocks/BlocksDrawer/index.js'
 import * as facelessUIImport from '@faceless-ui/modal'
+export { SetViewActions } from '../providers/Actions/SetViewActions/index.js'
```

Comment:
Should this be exported from /providers? I know its technically an element but it lives elsewhere

Reply:
I moved this to its own file and BlocksDrawer to a `fields.ts` as well, I think it makes sense that it's consistent to the file path